src/slack/events/botTagged.ts

```diff
@@ -0,0 +1,15 @@
+import { App } from '@slack/bolt';
+import { Middleware, SlackEventMiddlewareArgs } from '@slack/bolt';
+import logger from '../../logger';
+
+export const botTagged: Middleware<SlackEventMiddlewareArgs<'app_mention'>> = async ({ event, say }) =>{
+
+  try {
+    say(`
```

Comment:
Using `say` here is going to clutter the channel. Instead, let's use [`chat.postMessage`](https://api.slack.com/methods/chat.postMessage#arg_thread_ts) and leverage the `message_ts` value of the original message to reply in a thread.